tests/rt.sh

```diff
@@ -304,30 +304,32 @@ elif [[ $MACHINE_ID = acorn ]]; then
   PTMP=/lfs/h2/emc/ptmp
   SCHEDULER=pbs
 
-elif [[ $MACHINE_ID = gaea-c5 ]]; then
+elif [[ $MACHINE_ID = gaea ]]; then
 
-  module use /lustre/f2/dev/role.epic/contrib/C5/rocoto/modulefiles
+  module use /ncrc/proj/epic/rocoto/modulefiles
   module load rocoto
   ROCOTORUN=$(which rocotorun)
   ROCOTOSTAT=$(which rocotostat)
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
+  
   module load PrgEnv-intel/8.3.3
   module load intel-classic/2023.1.0
   module load cray-mpich/8.1.25
   module load python/3.9.12
-  module use /lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/modulefiles
+  module use /ncrc/proj/epic/spack-stack/modulefiles
+  module load gcc/12.2.0
   module load ecflow/5.8.4
-  ECFLOW_START=/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/ecflow-5.8.4/bin/ecflow_start.sh
+  ECFLOW_START=/ncrc/proj/epic/spack-stack/ecflow-5.8.4/bin/ecflow_start.sh
   ECF_PORT=$(( $(id -u) + 1500 ))
 
-  DISKNM=/lustre/f2/pdata/ncep/role.epic/C5/RT
+  DISKNM=/gpfs/f5/epic/world-shared/UFS-WM_RT
   QUEUE=normal
   COMPILE_QUEUE=normal
   PARTITION=c5
-  STMP=/lustre/f2/scratch
-  PTMP=/lustre/f2/scratch
+  STMP=/gpfs/f5/epic/scratch
```

Comment:
How is someone outside of the epic group supposed to run the RTs?

Reply:
@jkbk2004 can you comment on this? i thought we switched this...

Reply:
f5 scratch file system is maintained project-base. Users need to switch over to their own scratch project: e.g. /gpfs/f5/ncep/scratch